doc/sonic-reduce-disk-io/sonic-reduce-disk-io.md

```diff
@@ -0,0 +1,176 @@
+
+
+
+### Results with Optimizations
+Below are the disk writes in MiB per day after implementing the optimizations proposed above:
+
+| HWSKU | OS Version | Unoptimized | Optimizations A |  Optimizations B | Optimization C |
+| ----- | ---------- | ----------- | --------------- | ---------------- | -------------- |
+| SKU1  | Version_1  | 872.89      | 147.1           | 135.9            | 66             |
+|       | Version_2  | 667.81      | 273.3           | 55.8             | 81             |
+|       | Version_3  | 719.52      | 133.2           | 87.5             | 70.4           |
+| SKU2  | Version_1  | 627.81      | 99.1            | 23.5             | 62.9           |
+|       | Version_3  | 729.14      | 43.3            | 8                | 77             |
+| SKU3  | Version_1  | 762.63      | 237.3           | 175.3            | 50             |
+|       | Version_2  | 591.26      | 160.1           | 94.4             | 52             |
+|       | Version_3  | 643.14      | 96              | 206              | 59             |
+
+##### **Table 4**
+
+Where:
+Optimization A: User space modifications, journaling is enabled and **/tmp is mounted to disk**
+Optimization B: User space modifications, journaling is enabled and **/tmp is mounted to tmpfs**
+Optimization C: optimization A + **kernel journaling disabled**
+
+
+Here is the summarized table showing the reduction in disk writes for each optimization technique as a percentage of the unoptimized value:
+
+| **HWSKU** | **OS Version** | **Optimization A (%)** | **Optimization B (%)** | **Optimization C (%)** |
+|-----------|----------------|------------------------|------------------------|------------------------|
+| SKU1      | Version_1      | 83.11%                 | 84.43%                 | 92.44%                 |
+| SKU1      | Version_2      | 59.08%                 | 91.64%                 | 87.87%                 |
+| SKU1      | Version_3      | 81.49%                 | 87.84%                 | 90.22%                 |
+| SKU2      | Version_1      | 84.21%                 | 96.26%                 | 89.98%                 |
+| SKU2      | Version_3      | 94.06%                 | 98.90%                 | 89.44%                 |
+| SKU3      | Version_1      | 68.88%                 | 77.01%                 | 93.44%                 |
+| SKU3      | Version_2      | 72.91%                 | 84.03%                 | 91.21%                 |
+| SKU3      | Version_3      | 85.07%                 | 67.97%                 | 90.83%                 |
+
+<br>
+<br>
+
+![Reduction in disk writes as a percentage of writes in unoptimized SONiC images vs various optimization strategies](images/Reduction_Percentage.png)
+
+
+<br>
+<br>
+
+### Key Takeaways from Disk Write Comparisons Across Scenarios
+
+1. **Optimization A** consistently provides the highest reduction across most configurations, often exceeding 90%.
```

Comment:
Looking at the percentages above, doesn't optimization B reduce disk writes further than optimization A in all but one scenario?

Reply:
Hey good catch, that was supposed to say Optimization A. Will change.